scripts/check-and-run-apollo-codegen.sh

```diff
@@ -46,20 +47,28 @@ if [[ -x "$HOME/.nodenv/bin/nodenv" ]]; then
   eval "$("$HOME/.nodenv/bin/nodenv" init -)"
 fi
 
-if ! type "apollo-codegen" >/dev/null 2>&1; then
-  echo "Can't find apollo-codegen. Installing..."
-  install_apollo_codegen
-fi
+if [[ -s "$SRCROOT/node_modules/.bin/apollo-codegen" ]]; then
+  # If it's installed locally, use version build instead
+  set -x
 
-INSTALLED_APOLLO_CODEGEN_VERSION="$(get_installed_version)"
+  exec "$SRCROOT/node_modules/.bin/apollo-codegen" "$@"
```

Comment:
Do we not want to test the version of `apollo-codegen` here?
It could still be an incompatible version, couldn’t it?